### ISSUE TITLE
Allow setting of a global image registry override for the whole installer

### ISSUE THREAD

**Issue by mowies** [keptn/keptn#6043]
This is a task derived from #5456 and it depends on #6042.

It is tricky to set all the right parameters in helm to override where the different keptn image should be pulled from since in the current installer, image and repository are both in the same variable.

This could be improved by introducing separate `image` and `repository` variables together also with image pull secrets that can be customized globally.
Those parameters are already provided globally from our helper functions in the common library helm chart.

Lots of work for this was already done in #5456 so it should be used as inspiration and baseline for this ticket.
